docs/pages/protocol/overview.mdx

```diff
@@ -0,0 +1,49 @@
+# XMTP protocol overview
+
+XMTP is a decentralized messaging protocol that enables secure, end-to-end encrypted communication between blockchain addresses. This section covers the technical foundations and implementation details of how XMTP works under the hood.
+
+## Architecture fundamentals
+
+With XMTP, messages are routed through **[topics](/protocol/topics)**, which are unique addresses that identify conversation channels. Messages are packaged as **[envelope types](/protocol/envelope-types)** that contain the actual message data plus metadata for routing and processing.
+
+The protocol prioritizes **security** through advanced cryptographic techniques, delivering end-to-end encryption for both 1:1 and group conversations. Learn about XMTP's [security properties](/protocol/security) and how [wallet signatures](/protocol/signatures) authenticate users.
+
+## Group messaging mechanics
+
+For group conversations, XMTP introduces sophisticated state management:
+
+- **[Epochs](/protocol/epochs)** represent the cryptographic state of a group at any point in time. Each group operation (like adding members) creates a new epoch with fresh encryption keys.
```

Comment:
Epochs are a part of the MLS protocol, so I don't want to make it sound like XMTP invented this concept without acknowledging where it came from! Intents/cursors and topics/identity are things that XMTP added on top though.